common/locales/en/npc.json

```diff
@@ -41,7 +41,7 @@
     "autoAllocate": "Auto Allocate",
     "autoAllocateText": "If 'automatic allocation' is checked, your avatar gains stats automatically based on your tasks' attributes, which you can find in <strong>TASK > Edit > Advanced > Attributes</strong>. Eg, if you hit the gym often, and your 'Gym' Daily is set to 'Physical', you'll gain Strength automatically.",
     "spells": "Spells",
-    "spellsText": "You can now unlock class-specific spells. You'll see your first at level 11. Your mana replenishes 10 points per day, plus 1 point per completed ",
+    "spellsText": "You can now unlock class-specific spells. You'll see your first at level 11. Your mana replenishes 10 points per day, plus 1 point per completed <a target='_blank' href='http://habitica.wikia.com/wiki/Todos'>To-Do</a>",
```

Comment:
Oh and the rest of that line is outdated too - you also get mana from completed Dailies and positive Habits, and the amount of mana per day isn't always 10. Maybe use something like this instead:

`"spellsText": "You can now unlock class-specific skills. You'll see your first at level 11. You gain more mana each morning and when you accomplish tasks.",`